docs/puffer-preconf/puffer-preconf-intro.md

```diff
@@ -0,0 +1,179 @@
+- Additional redundancy and reliability
+- Timeline: [To be announced]
+
+## Permissionless & Neutral by Design 🌐
+
+Puffer Preconf is built to serve the entire Ethereum ecosystem, not just Puffer's own rollup (UniFi):
+
+### Why Trust Matters for L2 Integration
+
+We understand that integrating with a preconf service requires trust that the provider won't favor their own rollup or extract unfair value. Puffer Preconf is designed with strict neutrality guarantees:
+
+- **🔓 No Token Gating**: No Puffer governance token required to participate as validator, gateway, or L2
+- **📖 Open Protocol**: Fully open-source with transparent on-chain registry
+- **⚖️ Equal Treatment**: All L2s receive the same service level and economic terms
+- **🔐 Cryptographic Commitments**: Trust the math, not the operator—preconfs are cryptographically signed and slashable
+- **🌍 Multi-Gateway**: Multiple independent gateways (Gattaca, Google, etc.) prevent centralization
+- **🎯 Aligned Incentives**: Validators earn from all L2s, not just UniFi—more L2s = better validator economics
+
+### UniFi: Leading by Example
+
+UniFi, Puffer's own based rollup, uses Puffer Preconf to prove the technology works. But our success depends on ecosystem adoption—we're stronger when all L2s benefit from fast, secure preconfirmations.
+
+## What You Get 💪
+
+### Execution Preconfirmations
+
+Unlike simple inclusion preconfs, Puffer Preconf provides **execution preconfirmations**:
+
+**Inclusion Preconf** (Weak Guarantee):
+- Transaction will be included in a block
+- Final state may differ from expected (e.g., price slippage)
+
+**Execution Preconf** (Strong Guarantee):
+- Transaction will execute with exact state as promised
+- No surprises—users get exactly what they signed
+
+**Example**: A user swaps ETH for 2,400 USDC:
+- Inclusion preconf: Transaction included, but final price might be 2,350 or 2,450 USDC
+- Execution preconf: **Guaranteed** to execute at exactly 2,400 USDC
+
+### Economic Benefits for L2 Operators
+
+Traditional based rollup models force you to cede all transaction fees to L1 validators. Puffer Preconf introduces a **revenue sharing model**:
+
+- L2 operators retain a portion of preconf fees
+- Validators earn from providing preconf services
+- Infrastructure costs are shared equitably
+- In case of slashing, affected parties are compensated
+
+This makes based rollup economics viable for appchains and general-purpose L2s.
+
+### User Protection
+
+- **Refund mechanism**: Users who paid for failed preconfs can claim refunds
+- **Slashing protection**: Misbehaving validators lose stake, which is redistributed to affected users
+
+## Current Status & Roadmap 🗓️
+
+**Phase 1** (Current):
+- Gateway delegation model
+- Gattaca gateway live
+- Reward forfeit for missed proposals
+- OP-based rollup support
+
+**Phase 2** (Coming Soon):
+- Additional gateways including Google
+- Enhanced lookahead mechanism
+- Expanded rollup architecture support
```

Comment:
Added this one because it's mentioned in the Puffer Preconf notion page. I only mentioned it in a couple places for now.